script/show-sysinit.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import os.path
+import sys
+import subprocess
+
+if __name__ == "__main__":
+  print('digraph mimiker_modules {')
+  for folder in ['sys','mips']:
```

Comment:
Python has lots of great tools for processing text, I guess you could rewrite the above to something similar to:
```python
sysinit = re.compile('SYSINIT_ADD')
for source in map(os.listdir, ['sys', 'mips']):
  lines = filter(sysinit.search, file(source).readlines())
```